liquidctl/driver/kraken3.py

```diff
@@ -807,12 +803,26 @@ def check_unsupported_fw_version():
             self._write([0x30, 0x02, 0x01, self.brightness, 0x0, 0x0, 0x1, int(value_int / 90)])
             return
         elif mode == "static":
-            check_unsupported_fw_version()
-            data = self._prepare_static_file(value, self.orientation)
-            self._send_data(data, [0x02, 0x0, 0x0, 0x0] + list(len(data).to_bytes(4, "little")))
+            if _is_2023_fw_version2():
+                data = self._prepare_static_file_rgb16(value, self.orientation)
+                self._send_2023_data_fw2(
+                    data, [0x06, 0x0, 0x0, 0x0] + list(len(data).to_bytes(4, "little"))
+                )
+                # sending it twice is only required once after initialization
+                # the same behaviour is observed in manufacturer at init
+                # some soft of framebuffer swapping?
+                self._send_2023_data_fw2(
```

Comment:
There's probably no way to reliably track if we're here just after an init, right?

Otherwise, I think it LGTM.

Reply:
It pains me to say this, but I didn't found a way to track this.
When manufacturer software loads, it sends the image twice at load.
After changing to different image, it sends it twice.
I tried to find any pattern that would indicate that second attempt is needed, but I failed.